github_issue_label_guide.md

```diff
@@ -0,0 +1,91 @@
+# Jina Issue Labeling
+
+## Scope
+These guidelines serve as a primary document for explaining how GitHub issues are labeled in Jina repos. For community members reading this document, it should explain how a label was assigned to your issue, and what that label means. For Jina staff, it describes how to assign labels correctly.
+
+**Note:** These guideline only apply to the following repos; [Jina](https://github.com/jina-ai/jina), [Jina-hub](https://github.com/jina-ai/jina-hub), [Dashboard](https://github.com/jina-ai/dashboard), [docs](https://github.com/jina-ai/docs), [cloud-ops](https://github.com/jina-ai/cloud-ops),  [Jinabox.js](https://github.com/jina-ai/jinabox.js)
+
+## Why is labeling important?
+* Labels provide a good starting point for community members to contribute to Jina.
+* Labels provide status for both community members and Jina staff on ongoing workstreams and feature requests.
+* Labels provide an overview of the development status of Jina, highlighting areas of issue that need attention. They can be used as metrics of productivity and health of the project.
+
+
+## How labels are used in Jina repos.
+
+Labels added by community members are automatically assigned the label `needs-triage`. A Jina staff member should then read the issue and assign it to the correct labels.
+
+An issue added by a Jina staff member can skip triage and directly be assigned a type, priority, status and area label.
+
+All issues are assigned a type label. Some issues may be assigned a priority, status, and area label. **An issue should only ever have one type label, one status label, one priority label, and one area label.**
+
+This is an example of a correctly labelled issue
+![alt text](.github/images/github_issue_label.png)
+
+
+
+## How to assign: a step-by-step flow
+This aims to walk you through a standard triaging process.
+
+#### 1. Assign a type label:
+| Label | Description and action to be taken  |
```

Comment:
like maybe here we can also reiterate that only 1 label can be assigned

Reply:
Yeah, I think we will need to reiterate this a lot ;)